bread/contrib/triggers/urls.py

```diff
@@ -0,0 +1,46 @@
+import htmlgenerator as hg
+
+from bread import layout
+from bread.utils import quickregister
+from bread.views import AddView, EditView
+
+from .models import DataChangeTrigger, DateFieldTrigger, SendEmail
+
+urlpatterns = []
+quickregister(
+    urlpatterns,
+    DataChangeTrigger,
+    editview=EditView._with(
+        fields=[
+            hg.H4(
```

Comment:
Is this a field? or is it just a trick for adding something arbitrary to the layout?

Reply:
Yes, a trick :D
Similar to datatable columns: When a simple string is passed it denotes a field which is derived from the model. If it is a "complex" object, then it will be used directly.

It makes prototyping easier as we do not have to override ```get_layout``` everytime.